### DISCUSSION TITLE
Graph dataset without node features?

### DISCUSSION THREAD

**Discussion by scottshufe** [pyg-team/pytorch_geometric#5472] [Q&A] [answered]
Does PyG have graph dataset (single graph) with only structure information and no node features?
(It would be better if the dataset was used for link prediction task in literatures)

**Comment by EdisonLeeeee** [comment, marked as answer]
Hi @scottshufe I think [Polblogs](https://pytorch-geometric.readthedocs.io/en/latest/modules/datasets.html#torch_geometric.datasets.PolBlogs) dataset might be what you are looking for in PyG. You can also take a look at this [website](http://snap.stanford.edu/data/index.html), where different sizes of graph data are provided and mostly without node features.

**Reply by rusty1s** [reply]
Some of the `TUDataset` ones also don't come with features. It is also generally applicable to just don't use existing node features in the first place.

**Reply by scottshufe** [reply]
Thanks a lot, guys :)
